examples/add_picture_claim_from_idp_metadata.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Set picture claim from IdP picture metadata
+ *
+ * Flow: Complement token, Trigger: Pre Userinfo creation, Pre access token creation
+ *
+ * @param ctx
+ * @param api
+ */
+function add_picture_claim_from_idp_metadata(ctx, api) {
+  // return if picture already set
+  if (ctx.v1.claims.picture) {
```

Comment:
can you add the change already mentioned by yourself with (ctx.v1.claims && ctx.v1.claims.picture) ?

Reply:
Done